reactor-core/src/main/java/reactor/core/publisher/ContextPropagation.java

```diff
@@ -49,7 +49,11 @@ final class ContextPropagation {
 
 	static final Logger LOGGER;
 
-	static final boolean isContextPropagationAvailable;
+	// Note: If reflection is used for this field, then the name of the field should end with 'Available'.
+	// The preprocessing for native-image support is Spring Framework, and is a short term solution.
+	// The field should end with 'Available'. See org.springframework.aot.nativex.feature.PreComputeFieldFeature.
```

Comment:
I think this comment is wrong. The PR changes the field name to not end with `Availible` however, the comment says it should end with `Availible`.

Reply:
The comment is fine. It begins with "if reflection is used" and currently reflection is NOT used. So if we introduce reflection here, we can rename it to end with "Available".